### ISSUE TITLE
Hidden prerequisite docker-compose (in correct version)

### ISSUE THREAD

**Issue by PhilippMundhenk** [GENIVI/iot-event-analytics#90]
When following the prerequisites & tldr, I receive the following message from the IoT Event Analytics (Extension):
```
Cannot parse docker-compose version string "docker-compose version 1.17.1, build unknown"
```
The `1.17.1, build unknown` is the current version on Ubuntu 18.04.5.

When using the current '1.29.2, build 5becea4c' from GitHub, it works file.

**Comment by llakie** [comment]
That issue occurs, since we are parsing the given version string and we are expecting a hash as last argument (not "unknown").
We will make the parser more generous, since we are only interested in the version string. Thanks for tracking that down.

**Comment by llakie** [comment]
Will be fixed in #100

**Comment by llakie** [comment]
@PhilippMundhenk can you please check?
